docs/index.md

```diff
@@ -1,27 +1,20 @@
 # HIP documentation
 
 The Heterogeneous-computing Interface for Portability (HIP) API is a C++ runtime
-API and kernel language that lets developers create portable applications for AMD
-and NVIDIA GPUs from single source code.
+API and kernel language that lets developers create portable applications running
+in heterogeneous systems, using CPUs and AMD GPUs and NVIDIA GPUs from a single source code.
```

Comment:
in heterogeneous systems using CPUs, and AMD GPUs or NVIDIA GPUs, from a single source code.